question/bank/deletequestion/delete.php

```diff
@@ -77,7 +77,7 @@
                 $questionid = (int)$questionid;
                 question_require_capability_on($questionid, 'edit');
                 if (questions_in_use(array($questionid))) {
-                    $DB->set_field('question', 'hidden', 1, array('id' => $questionid));
+                    $DB->set_field('question_versions', 'status', 1, ['questionid' => $questionid]);
```

Comment:
Maybe we need something similar in line 61 https://github.com/catalyst/moodle-MDL-70329/pull/259/files#R61